github/libp2p.yml

```
@@ -4141,6 +4141,8 @@ repositories:
     delete_branch_on_merge: true
     description: Interoperability tests for libp2p Implementations
     files:
+      .github/js_pull_request_template.md:
```

If we want to make the template a default in the repo, we should name it `pull_request_template.md` in the target repo. As per https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

```suggestion
      .github/pull_request_template.md:
```